docs/user-manual/en/broker-balancers.md

```diff
@@ -0,0 +1,156 @@
+* `SNI_HOST` is the hostname indicated by the client in the SNI extension of the TLS protocol;
+* `SOURCE_IP` is the source IP address of the client;
+* `USER_NAME` is the username indicated by the client.
+
+## Pools
+The pool is a group of brokers and checks periodically their state.
+Including the local target broker in the pool allows broker hosting the broker balancer to accept incoming client connections. 
+By default, a pool doesn't include the local target broker, to include it the `local-target-enabled` parameter must be `true`.
+A pool becomes ready when the minimum number of ready targets defined by the `quorum-size` parameter is reached.
+This behaviour avoids weird distribution at startup or after a restart.
+There are two pool types: [discovery pool](#discovery-pool) and [static pool](#static-pool).
+
+### Discovery Pool
+The discovery pool uses a [discovery group](clusters.md#discovery-groups) to discover the target brokers to add.
+Let's take a look at a discovery pool example from broker.xml that uses a discovery group:
+```xml
+<pool>
+    <discovery-group-ref discovery-group-name="dg1"/>
+</pool>
+```
+
+### Static Pool
+The static pool uses a list of static connectors to define the target brokers to add.
+Let's take a look at a static pool example from broker.xml that uses a list of static connectors:
+```xml
+<pool>
+    <static-connectors>
+        <connector-ref>connector1</connector-ref>
+        <connector-ref>connector2</connector-ref>
+        <connector-ref>connector3</connector-ref>
+    </static-connectors>
+</pool>
+```
+
+### Defining pools
+A pool is defined by the `pool` element that includes the following items:
+* the `username` element defines the username to connect to the target broker;
+* the `password` element defines the password to connect to the target broker;
+* the `check-period` element defines the often to check the target broker;
+* the `quorum-size` element defines the minimum numbers of ready tagrets;
+* the `local-target-enabled` element defines whether the pool has to include a local target;
+* the `static-connectors` element defines a list of static connectors used by the [static pool](#static-pool);
+* the `discovery-group` element defines the [discovery group](clusters.md#discovery-groups) used by the [discovery pool](#discovery-pool).
+
+Let's take a look at a pool example from broker.xml:
+```xml
+<pool>
+    <check-period>1000</check-period>
+    <local-target-enabled>true</local-target-enabled>
+    <quorum-size>2</quorum-size>
+    <static-connectors>
+        <connector-ref>connector1</connector-ref>
+        <connector-ref>connector2</connector-ref>
+        <connector-ref>connector3</connector-ref>
+    </static-connectors>
+</pool>
+```
+
+## Policies
+The policy define how to select a broker from a pool. The included policies are:
+* `FIRST_ELEMENT` to select the first target broker from a pool;
```

Comment:
the _ELEMENT seems odd? FIRST on it's own or HEAD (but that implies a list). Can you add a one liner when FIRST would make sense, ROUND_ROBIN with a single entry in a pool would behave the same.

Reply:
I can't find a better name so I used `FIRST_ELEMENT` to be consistent with the similar client load balancing policy: org.apache.activemq.artemis.api.core.client.loadbalance.FirstElementConnectionLoadBalancingPolicy